src/caffe/proto/caffe.proto

```diff
@@ -1231,4 +1231,11 @@ message PReLUParameter {
   optional FillerParameter filler = 1;
   // Whether or not slope paramters are shared across channels.
   optional bool channel_shared = 2 [default = false];
+  // Whether or not to constrain the negative slope between min_neg_slope and
+  // max_neg_slope.
+  optional bool constrain_neg_slope = 3 [default = true];
```

Comment:
Since there is no migration procedure for those who have not specified `constrain_neg_slope` yet, I think that the default should be `false`.  Otherwise this silently changes the behavior of any existing nets, and this can be really hard to debug.